packages/aws-fargate/package.json

```diff
@@ -10,7 +10,8 @@
   "main": "src/index.js",
   "files": [
     "src",
-    "esm-loader.mjs"
+    "esm-loader.mjs",
```

Comment:
issue: Putting this here. We need to prepare the autotrace webhook. You need to prepare the changes in a PR, deploy our npm packages and then immediately deploy the webhook with the latest version.

Reply:
Discussed offline, we will do this update once after releasing this fix